packages/vector/src/services/chroma-provider.ts

```
@@ -0,0 +1,123 @@
+    await collection.delete({
+      ids: options.ids,
+    })
+  }
+  
+  private async getCollection(name: string): Promise<Collection> {
+    try {
+      return await this.client.getCollection({
+        name,
+        embeddingFunction: this.embedder,
+      })
+    } catch (error: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)